core/controllers/admin.py

```diff
@@ -52,6 +80,31 @@ def test_super_admin(self, **kwargs):
     return test_super_admin
 
 
+def check_and_update_config_role(old_list, new_list, role):
+    """Changes roles for users in old_list and new_list. If user belongs to
+    new_list but not in old list, its role is changed to specified role. If a
+    user belongs to old list but not new list, its role is changed to
+    exploration editor.
+
+    Args:
+        old_list: list(str). Existing list of usernames with given
+            role.
+        new_list: list(str). Updated list of usernames with given role.
+        role: str. The role to be used for users in these lists.
+    """
+    for username in old_list:
```

Comment:
replying to:
`I don't understand your last bullet point,sorry. Could you please elaborate?`

I meant migration job expects releaser to adjust lists so as to have one to one mapping with the roles in new system.   So, will the releaser eventually readjust the list to its origingal condition (one where there is same username in multiple lists).